kitchen.dokken.yml

```
@@ -1,8 +1,9 @@
 driver:
   name: dokken
   privileged: true  # because Docker and SystemD/Upstart
-  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  chef_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
```

Please keep this using `current`
```suggestion
  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
```

---

`current` can potentially test against a version of Chef Infra Client that is considered unstable. I'd rather not be testing against unstable/unreleased Chef Infra client versions. `Latest` appears to be a good compromise in terms of getting the newest version possible and also testing against a stable version.

---

This catches any potential upstream issues/fixes before a stable release that happens upstream and this has been a standard we've been using globally on all of our cookbooks. It should be stable enough for testing IMHO.

---

What you're proposing unnecessarily ties the release of chef-splunk to the releases of other cookbooks. This setting of `current` was the root cause of the initial Chef 17 incompatibility issues. The version of Chef Infra Client that GH Actions was pulling in and testing was different than the version tested when the PR was submitted. The `current` setting allows for "whatever version happens to be on the test machine" at the time of test. In this case, the version was 17.0.150 that was never released -- except only in Docker Hub -- and GH Actions tested with the 17.0.242 version that was an official released version. Setting this as `current` creates an unstable development release cycle.